src/main/webapp/app/iris/settings/iris-settings-update/iris-settings-update.component.html

```diff
@@ -27,6 +27,13 @@ <h3 jhiTranslate="artemisApp.iris.settings.subSettings.chatSettings"></h3>
                 (onChanges)="isDirty = true"
             />
         </div>
+        @if (settingsType === COURSE) {
+            <h4 class="form-label mt-3" jhiTranslate="artemisApp.iris.settings.subSettings.lectureChat.title"></h4>
+            <div class="form-check form-switch">
+                <input class="form-check-input" type="checkbox" id="lectureChatSwitch" [disabled]="!isAdmin" [(ngModel)]="lectureChat" />
+                <label class="form-check-label" for="lectureChatSwitch" jhiTranslate="artemisApp.iris.settings.subSettings.lectureChat.lectureChatIngestion"></label>
+            </div>
```

Comment:
It may be helpful to describe more what the lecture ingestion is to the instructor